packages/node_modules/cerebral/src/views/angularjs/connect.js

```diff
@@ -1,32 +1,47 @@
-function connect(dependencies, extendedCtrl) {
-  if (typeof extendedCtrl === 'function') {
+function connect(dependencies, ...args) {
+  let displayName = 'Unknown'
+  if (args.length > 2) {
+    throw new Error(
+      `connect() expects 2 or 3 arguments, got ${args.length + 1}.`
+    )
+  }
+  let displayNameIdx = args.findIndex(a => typeof a === 'string')
```

Comment:
findIndex is not supported in IEs? Is it worth to use it just for flexible argument position?